.yarn/patches/@metamask-keyring-controller-npm-17.2.0-667fdd6864.patch

```diff
@@ -1,5 +1,5 @@
 diff --git a/package.json b/package.json
-index 5a6217eaed16fdfe7f1ad693871f85320bd6b421..69bdf1a9155497e37fc58db7bbc74597fd543535 100644
+index 1caaa7e433c4597d2554227536e87f16ca9ea117..29c11adb88149d3d6aef5abc049c19dedab71a91 100644
 --- a/package.json
 +++ b/package.json
```

Comment:
Whats up with this patch change?

Reply:
I kept the original patch, but updated it to match the bumped version of keyring-controller